images/linux/scripts/SoftwareReport/SoftwareReport.CachedTools.psm1

```diff
@@ -29,11 +29,32 @@ function Get-ToolcacheGoVersions {
 }
 
 function Get-ToolcacheBoostVersions {
+    $Name = "Boost"
     $toolcachePath = Join-Path $env:AGENT_TOOLSDIRECTORY "boost"
     if (-not (Test-Path $toolcachePath)) {
         return @()
     }
-    return Get-ChildItem $toolcachePath -Name | Sort-Object { [Version]$_ }
+
+    $BoostVersions = Get-ChildItem $toolcachePath -Name | Sort-Object { [Version]$_ }
+    $ToolInstances = $BoostVersions | ForEach-Object {
+        $VersionEnvVar = $_.replace(".", "_")
+        return @{
+            Version = $_
+            Architecture = "x64"
+            "Environment Variable" = "BOOST_ROOT_${VersionEnvVar}"
+
+        }
+    }
+    $Content = $ToolInstances | New-MDTable -Columns ([ordered]@{
+    Version = "left";
+    Architecture = "left";
+    "Environment Variable" = "left"
+    })
+
+    $markdown = New-MDHeader $Name -Level 4
+    $markdown += New-MDParagraph -Lines $Content
```

Comment:
Almost done.
I think we don't these two lines here. Otherwise, we will have duplicate of header. Let's just `return $Content`

Currently, result looks like
![image](https://user-images.githubusercontent.com/16715858/100201748-76433e80-2f11-11eb-90b6-910a4907fb84.png)

Reply:
Shouldn't we keep line 55 as `$markdown = New-MDParagraph -Lines $Content` so that the table actually renders?

Reply:
I figured out why this was happening, updated.